### DISCUSSION TITLE
Example Rnote Files?

### DISCUSSION THREAD

**Discussion by ahcox** [flxzt/rnote#909] [Show and tell]
Does anyone have any large Rnote files to share?

Sketches with tens to hundreds of thousands of strokes?

Long files of handwritten notes, for example from studying some STEM topic deeply?

Thanks for sharing if you do!

**Comment by mahdi4261** [comment]
> Sketches with tens to hundreds of thousands of strokes?

How would I know how many strokes my note contain?

**Reply by Doublonmousse** [reply]
With the dev mode activated, it's shown in the visual debugging option of the dev menu.


![image](https://github.com/user-attachments/assets/c98a8c70-76ef-4e1f-a997-467a1d90da10)

The real question I have is why though? What's the purpose of having a threshold on the number of strokes

**Reply by mahdi4261** [reply]
Thanks, I didn't know that. Actually, there is no need for that. I just got curios after reading his message, if it was possible.

**Comment by Kreijstal** [comment]
I guess no one had rnote files to share? wild

**Comment by Doublonmousse** [comment]
There are some (short) rnote files here https://github.com/flxzt/rnote/tree/main/misc/screenshots-pages
